tests/app/authentication/test_authenticator.py

```diff
@@ -94,8 +94,7 @@ def test_no_user_when_session_has_expired(self):
                 self.assertIsNone(user)
                 self.assertIsNone(cookie_session.get(USER_IK))
 
-
-    def test_valid_user_extends_session_expiry(self):
+    def test_valid_user_does_not_extend_session_expiry_when_time_close(self):
```

Comment:
```suggestion
    def test_valid_user_does_not_extend_session_expiry_when_expiry_less_than_60_seconds_different(self):
```
I don't think this really makes sense. Maybe like your other one?. I guess its less than or equal to though.

Or maybe
``` python
def test_valid_user_does_not_extend_session_expiry_when_expiry_not_greater_than_60_seconds_different(self):
```
Or even change the code to `>= 60`. 🤷‍♂️ 
Up to you.